examples/image-vis-type-example.ipynb

```diff
@@ -14,7 +14,8 @@
     "\n",
     "```\n",
     "pip install mapboxgl\n",
-    "pip install Pillow numpy\n",
+    "pip install imread\n",
+    "pip install numpy\n",
```

Comment:
same here, numpy will be installed by default when installing `mapboxgl-jupyter`